app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy(command: str) -> None:
+    command = command.split()
+    if command[1] == command[2] or command[0] != "cp":
+        return
```

Comment:
Why you left empty return? Provide at least either `print` or some `error` message.

Reply:
Ok, I just thought it would be better. In the task written, it must do nothing, and Danylo in the lecture said it`s ok to use bare return.

Reply:
In that case it would be better to use reverse logic and complete copy if
`command[1] != command[2] and command[0] == "cp"`
don't you agree?)